### PR TITLE
app menu unit test

### PR DESCRIPTION
**Pull request by vrbsm**
## Description
app menu unit test

## QA Checklist
- [x] Unit-Tests
- [ ] Automation-Tests

Attach unit & spectron tests results in PDF format against the above task lists for this branch
[Symphony Electron Test Result.pdf](https://github.com/symphonyoss/SymphonyElectron/files/2990629/Symphony.Electron.Test.Result.pdf)

### CODE REVIEW

File: spec/appMenu.spec.ts [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+describe('app menu', () => {
+    let appMenu;
+    const updateUserFnLabel = 'updateUserConfig';
+    const item = {
+        checked: true,
+    };
+
+    const findMenuItemBuildWindowMenu = (menuItemLabel: string) => {
+        // tslint:disable-next-line:no-shadowed-variable
+        return appMenu.buildWindowMenu().submenu.find((item) => {
```
Comment (KiranNiranjan) [inline spec/appMenu.spec.ts]: @vrbsm You can just use a different name like `menuItem` instead of disabling `tslint`

File: spec/appMenu.spec.ts [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+            return item.label === menuItemLabel;
+        });
+    };
+
+    const findMenuItemBuildHelpMenu = (menuItemLabel: string) => {
+        // tslint:disable-next-line:no-shadowed-variable
+        return appMenu.buildHelpMenu().submenu.find((item) => {
```
Comment (KiranNiranjan) [inline spec/appMenu.spec.ts]: same here

File: spec/appMenu.spec.ts [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+            return item.label === menuItemLabel;
+        });
+    };
+
+    const findHelpTroubleshootingMenuItem = (value: string) => {
+        const helpMenuItem = findMenuItemBuildHelpMenu('Troubleshooting');
+        // tslint:disable-next-line:no-shadowed-variable
+        return helpMenuItem.submenu.find((item) => {
```
Comment (KiranNiranjan) [inline spec/appMenu.spec.ts]: same here

**Comment by VishwasShashidhar** [comment]
@vrbsm Please resolve the merge conflicts.

**Pull request merged**